modules/missenaBidAdapter.js

```diff
@@ -0,0 +1,94 @@
+import * as utils from '../src/utils.js';
```

Comment:
Can you please import only the needed utility functions here? We have changed how all adapters import from utils